app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/attachments_list.scala.html

```diff
@@ -52,9 +59,18 @@
                                     </div>
                                 </td>
                             }
+
                             <td id="@{id}-row-@{index}-date" class="w-15">
                                 @Dates.format(file.timestamp)
                             </td>
+
+                            @if(showRemoval &&
+                                caseRef.isDefined &&
+                                request.hasPermission(Permission.REMOVE_ATTACHMENTS)) {
```

Comment:
Should this link always be here for every attachment? The ticket mentions about an attachment that's already published should replace the remove link with some text that says 'Published'

Reply:
After a talk through about this - it's not part of the AC at the moment for this ticket, so not an issue. In the future it could possibly be derived from a combo of existing public flag on the attachment and case status of completed (to hide remove link)